man/rhsm.conf.5

```diff
@@ -181,6 +181,11 @@ pluginConfDir
 .RS 4
 The directory to search for plug-in configuration files
 .RE
+.PP
+inotify
+.RS 4
+Inotify is used for monitoring changes in directories with certificates. Currently only /etc/pki/consumer directory is monitored by rhsm.service. When this directory is mounted using network file system without support of inotify notification (e.g. NFS), then it is strongly recommended to disable inotify. When inotify is disabled, then periodical directory polling is used.
```

Comment:
If this is the same text as the comment in the conf file, then the previous comment should cover this also.